style/addressing.mss

```diff
@@ -37,6 +49,12 @@
       text-halo-radius: @standard-halo-radius * 1.25;
       ["addr_unit" != null]["addr_housenumber" = null] {
         text-name: [addr_unit];
+        ["addr_flats" != null] {
+          text-name: [addr_unit] + "\n" + [addr_flats];
```

Comment:
Rendering just a unit number plus the flats numbers could be confusing

Reply:
Should addr_unit be dropped here?

Reply:
It seems like a tagging error. Could you download some data from overpass-turbo and see what it looks like? Either the unit or the flats should be dropped, but I'm not sure which idea is best. 

You could use this query in overpass-turbo or JOSM: 

```
[out:xml][timeout:90];
(
  node["addr:unit"]["addr:flats"];
);
(._;>;);
out meta;
```

Reply:
As discussed in the comments here, this should be resolved now.